src/editors/playgroundController.ts

```diff
@@ -237,6 +244,36 @@ export default class PlaygroundController {
     return this._createPlaygroundFileWithContent(content);
   }
 
+  async createPlaygroundForCreateCollection(
+    element: ConnectionTreeItem | DatabaseTreeItem
+  ): Promise<boolean> {
+    const dataService = this._connectionController.getActiveDataService();
+    let content = playgroundCreateCollectionTemplate;
+
+    if (dataService) {
+      const adminDb = dataService.client.client.db('admin');
+      const buildInfo = await adminDb.command({ buildInfo: 1 });
+
+      if (buildInfo) {
+        const serverVersion = buildInfo.version.replace(/-.*$/, '');
+
+        if (semver.satisfies(serverVersion, '>= 5.0')) {
```

Comment:
```suggestion
        const serverVersion = buildInfo.versionArray[0];

        if (serverVersion >= 5) {
```

since `versionArray` is the machine-accessible variant of `version`?

Reply:
Replaced version with versionArray.